examples/torque_driven_ocp/save_and_load_cube.py

```diff
@@ -21,7 +21,9 @@ def run_and_save_ocp():
 if __name__ == "__main__":
     run_and_save_ocp()
 
-    ocp, sol = OptimalControlProgram.load(biorbd_model_path="cube.bioMod", name="cube_ocp_sol.bo")
+    ocp, sol = OptimalControlProgram.load(
+        biorbd_model_path=["cube.bioMod", "cube.bioMod", "cube.bioMod"], name="cube_ocp_sol.bo"
+    )
```

Comment:
This example fails. The pickle get ride of some information of the Penalty class. This is not picked by the test, because on the test made on pendulum, there is no constraints nor objectives

Reply:
I think we should have a test of the copy for all the tests (without the solve, but with the deep_assert)